tests/integration/crunchers/test_cruncher_constant_ratio.py

```diff
@@ -117,6 +120,9 @@ def test_relationship_usage(self, unequal_df, test_downscale_df):
             test_downscale_df.timeseries().columns.values.squeeze(),
         )
 
+        # Test we can append the results correctly
+        test_downscale_df.append(res)
```

Comment:
I would suggest making this test slightly more robust by checking that the data actually stayed in the append dataframe and didn't explode. That could be done by adding something like

```python
res_from_append = test_downscale_df.filter(whatever_kwargs_required_to_get_res_back_out)

# can't remember what the pyam testing function is but hope the idea makes sense...
pyam.testing.compare_iamdfs(res, res_from_append)

# you could also test that extra columns went in as expected e.g.
# might require some indexing here as you might want different values at different points depending on how 
# the auto-infilling is meant to work
test_downscale_df[extra_col] == expected_result  
```

Reply:
I've added the comparison between the filtered and the output here. I check for the column genuinely being added to the original directly after it is added.

Reply:
I meant more like https://github.com/znicholls/silicone/pull/84, just to make sure the appending hasn't exploded (there's also the question of what should happen if you have more than one value in the extra column)